docker/vmware/Pipfile

```diff
@@ -1,14 +1,13 @@
-    [[source]]
-    url = "https://pypi.org/simple"
-    verify_ssl = true
-    name = "pypi"
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
 
-    [packages]
-    pyVim = "*"
-    pyVmomi = "*"
-    vsphere-automation-sdk = {file = "https://github.com/vmware/vsphere-automation-sdk-python/archive/refs/tags/v7.0.3.0.tar.gz"}
```

Comment:
are we able to add `git+https://github.com/vmware/vsphere-automation-sdk-python.git@v8.0.0.1` here similiar to how this was done in the past instead of to the dockerfile?

Reply:
we tried it doesn't work, the installation via pipenv fails with python 3

Reply:
at the end of the day this gets installed using a requirements.txt file, not pipenv- thats how our build works under the hood. What syntax did you use to add this to the pipfile?

Reply:
vsphere-automation-sdk = {file = "https://github.com/vmware/vsphere-automation-sdk-python/archive/refs/tags/v8.0.0.1.tar.gz"}

vsphere-automation-sdk-python = {git = "https://github.com/vmware/vsphere-automation-sdk-python.git", ref = "v7.0.1.0" }

pipenv install git+https://github.com/vmware/vsphere-automation-sdk-python.git#egg=vSphere-Automation-SDK

and more.. with different versions, all gave the same error as mention here: https://github.com/vmware/vsphere-automation-sdk-python/issues/302
I searched it more in google, and also consulted with ilan, but maybe there is a way I'm not aware of?

Reply:
this is happening when running pipenv lock?

Reply:
yes